makefiles/iodine.mk

```diff
@@ -0,0 +1,43 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS      += iodine
+IODINE_VERSION := 7.0-df49fd6
+DEB_IODINE_V   ?= $(IODINE_VERSION)
+
+IODINE_COMMIT  := df49fd6f3d9030662374bfbdcca3e74327084f5e
```

Comment:
Please change this to calculate the version like https://github.com/ProcursusTeam/Procursus/blob/main/makefiles/libimobiledevice.mk#L7